src/components/HomeTopics.vue

```diff
@@ -17,19 +20,55 @@ onMounted(() => {
 const goToPage = (page) => {
   window.location.href = page
 }
+
+const url_highlighted_topics = config.website.url_highlighted_topics
 </script>
 
 <template>
-  <div v-if="topicsData.length" class="datagouv-components fr-mt-5w">
-    <h2 class="subtitle subtitle--uppercase text-align-left">Thématiques</h2>
-    <div class="fr-grid-row fr-grid-row--gutters">
-      <div v-for="topic in topicsData" class="fr-col-12 fr-col-md-6">
-        <div
-          class="topic-card"
-          @click="goToPage('/datasets?topic=' + topic.id)"
-        >
-          <div class="topic-title">{{ topic.name }}</div>
-          <div class="topic-description">{{ topic.description }}</div>
+  <div>
+    <div v-if="type_highlighted_topics == 'card'">
+      <br />
+      <div style="display: flex; flex-wrap: wrap">
+        <span v-for="topic in topicsData" v-bind:key="topic.id">
+          <DsfrCard
+            style="width: 350px; margin-right: 30px; margin-bottom: 50px"
+            alt-img="altImg"
+            :description="topic.extras['udata-front-kit:short_description']"
+            :img-src="topic.extras['udata-front-kit:topic_image']"
+            :link="'/' + url_highlighted_topics + '?topic=' + topic.id"
+            :title="topic.name"
+          />
+        </span>
+      </div>
+    </div>
```

Comment:
This should be a component. And ideally it should be the same component as below (tile) with conditional styles/markup based on type, not totally different markups (some things seem common).